src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -37,11 +38,18 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
             this.extensionHost.VisualStateManager.BindingStateChanged += this.SolutionTracker_ActiveSolutionChanged;
             this.solutionTracker.ActiveSolutionChanged += this.SolutionTracker_ActiveSolutionChanged;
             this.CalculateSolutionBinding();
+            this.OnAnalyzersChanged();
```

Comment:
@eyalantzMS This is a hack here, I raise the `AnalyzersChanged` event the first time a solution is being opened. Shouldn't `this.solutionTracker.ActiveSolutionChanged` be triggered?